README.md

```diff
@@ -1,16 +1,21 @@
 
 # Axe + Azure Pipelines: Automate accessibility testing in your CI builds
 
-Samples demonstrating how to use [axe](https://www.deque.com/axe/) for automated accessibility testing of websites in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds.
-
-Automated accessibility tests can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing. We recommend [Accessibility Insights](https://accessibilityinsights.io/docs/en/web/overview), a free and open source dev tool that walks you though assessing a website for 100% [WCAG 2.0 AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&versions=2.0&levels=aaa) coverage.
+This project contains sample projects (see next section [Available samples](#available-samples)) demonstrating how to implement automated accessibility testing in [Azure Pipelines](https://azure.microsoft.com/en-us/services/devops/pipelines/) builds using [axe-core](https://github.com/dequelabs/axe-core), the same accessibility scanning engine used in [Accessibility Insights for Web](https://github.com/microsoft/accessibility-insights-web).

```

Comment:
For AI4Web, I think based on past discussions that Fer's preference was for the link to go to https://accessibilityinsights.io/docs/en/web/overview rather than github